seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/config/JdbcConfig.java

```diff
@@ -25,74 +27,74 @@
 
 public class JdbcConfig implements Serializable {
 
-    public static final String URL = "url";
+    public static final Option<String> URL = Options.key("url").stringType().noDefaultValue().withDescription("url");
 
-    public static final String DRIVER = "driver";
+    public static final Option<String> DRIVER =  Options.key("driver").stringType().noDefaultValue().withDescription("driver");
 
-    public static final String CONNECTION_CHECK_TIMEOUT_SEC = "connection_check_timeout_sec";
+    public static final Option<Integer> CONNECTION_CHECK_TIMEOUT_SEC = Options.key("connection_check_timeout_sec").intType().noDefaultValue().withDescription("connection check time second");
```

Comment:
In the doc, connection_check_timeout_sec have default value : 30
![image](https://user-images.githubusercontent.com/32387433/201561105-350f4260-dd4d-4dc1-848f-8206d168c7bb.png)

Reply:
OK,done

Reply:
<img width="918" alt="image" src="https://user-images.githubusercontent.com/32387433/201615011-da241202-9a5f-4015-ade4-8fc537b15607.png">
Modify this value too, use `CONNECTION_CHECK_TIMEOUT_SEC.defaultValue()` replaced.